### DISCUSSION TITLE
Consideration to add ridesharing

### DISCUSSION THREAD

**Discussion by BENGMN** [mudita/MuditaOS#4100] [Ideas]
Hello, is there any consideration to add ridesharing applications to the phone?

**Comment by freewheeler** [comment]
With the phone we can access rideshare sites. A simple, free example: http://liftcentrale.tripod.com

**Comment by ShantanuJoshi** [comment]
Thinking about implementing this. We can utilize the uber api's directly but we'll need to find a way to setup server tokens for each device. Maybe the onboard of the app asks a user to register w/ uber and get a server token + rider info added before we show fields to enter location addr & destination addr
